Server.js

```diff
@@ -44,6 +44,13 @@ app.all('*', function(req, res, next) {
 
 require('./api/V1')(app);
 
+// Add file processing API.
+var fileProcessingApp = express();
+fileProcessingApp.use(bodyParser.urlencoded({ extended: false }));
+require('./api/fileProcessing')(fileProcessingApp);
+http.createServer(fileProcessingApp).listen(config.fileProcessing.port);
+log.info('Starting file processing on', config.fileProcessing.port);
```

Comment:
What prevents the fileProcessing API from also being served on the normal port?

Reply:
I had it on a different port so it could only be accessible from the CCL servers.
If we want it open on the normal port we will need to add an authentication process and LeoFS to also be accessible in some way.